### ISSUE TITLE
Are formats available in vue 2?

### ISSUE THREAD

**Issue by adam-torok** [gruhn/vue-qrcode-reader#424]
Hey, basically what I'm trying to achieve, is to scan barcodes (ean_13, ean_8) in vue2 setup

I'm currently using 3.1.8 but I can only manage to read QR codes.

` 
 <v-row v-if="scanMode">
        <v-col cols="12">
            <qrcode-stream
                :constraints="{ facingMode }"
                :track="paintCenterText"
                :formats="formats"
                @detect="onDetect"
                @error="onError">
            </qrcode-stream>
        </v-col>
    </v-row>
`

In the data, here is my formats array

`formats: [
                'ean_13',
                'ean_8',
                'qr_code'
            ],`

So basically my question is, is it supported? 
Thanks

**Comment by adam-torok** [comment]
more formats has been added to the vue2 version so I close this
